test/testdata/nakedret.go

```diff
@@ -1,11 +1,15 @@
 //golangcitest:args -Enakedret
 package testdata
 
+import "fmt"
+
 func NakedretIssue() (a int, b string) {
 	if a > 0 {
-		return
+		return // want "naked return in func `NakedretIssue` with 33 lines of code"
 	}
 
+	fmt.Println("nakedret")
```

Comment:
@ldez were these `fmt.Println()` updates intentional?

should it still test the edge case of 31 lines, by removing two `// ...` lines below?

Reply:
Yes the println is intentional, because I wanted to have at least one std import.